experimental/adaptive-dialog/csharp_dotnetcore/02.multi-turn-prompt/AdapterWithErrorHandler.cs

```diff
@@ -2,35 +2,41 @@
 // Licensed under the MIT License.
 
 using System;
+using System.IO;
 using Microsoft.Bot.Builder;
-using Microsoft.Bot.Builder.Dialogs;
 using Microsoft.Bot.Builder.Dialogs.Debugging;
-using Microsoft.Bot.Builder.Dialogs.Declarative.Resources;
+using Microsoft.Bot.Builder.LanguageGeneration;
 using Microsoft.Bot.Builder.Integration.AspNet.Core;
 using Microsoft.Bot.Connector.Authentication;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.Logging;
+using ActivityBuilder = Microsoft.Bot.Builder.Dialogs.Adaptive.Generators.ActivityGenerator;
```

Comment:
> ActivityBuilder = Microsoft.Bot.Builder.Dialogs.Adaptive.Generators.ActivityGenerator [](start = 5, length = 86)

Why the rename of this class?  If this is a better name we should rename it in the SDK

Reply:
This is to avoid conflicts across LG and adaptive namespace.Dong has already fixed this but this sample is not using those packages. Will do another pass post Ignite.